catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -207,9 +210,14 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    private String buildDerivedImageTitle(String title, String qualifier) {
+    protected String buildDerivedImageTitle(String title, String qualifier) {
         String rootFileName = FilenameUtils.getBaseName(title);
-        return String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier, rootFileName, JPG);
+
+        return (StringUtils.isNotBlank(rootFileName) &&
```

Comment:
`StringUtils.isAlphanumeric(rootFileName)` is a much simpler test for both these conditions. https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isAlphanumeric(java.lang.CharSequence)